scalableminds/webknossos
# Check if task team is allowed for dataset

- only get tasks that are accessible to your team
### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create tasks for a dataset with no assigned team
- => no task available for you
- add allowed teams to dataset
- => task should be available to you

### Issues:
- fixes #4426 

------
